Makefile

```diff
@@ -4,7 +4,7 @@ PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 UNITDIR ?= $(PREFIX)/lib/systemd/system
-CSSLIBDIR ?= $(PREFIX)/lib/container-storage-setup
+CSSLIBDIR ?= $(PREFIX)/share/container-storage-setup
```

Comment:
library is usually used for storing executables, which I think the rpmlint is complaining about. Since we are just shipping scripts, then putting them into /usr/share seems fine.  I think systemd just ignores the error.